src/commands/AltCommand.java

```diff
@@ -42,6 +42,10 @@ public void execute(String[] input, SimulationManager simulationManager) {
             return;
         }
 
-        System.out.println(simulationManager.getCurrentStateString());
+        if (simulationManager.isDiffMode()) {
+            System.out.println(simulationManager.getCurrentStateDiffString(1));
```

Comment:
I don't think this is correct. If we `step 3` then do an alt. It's not an alt of the last step, but an alt of `step 3`, so another 3 step alt.

Reply:
Good catch!